packages/canary/src/components/treeview.tsx

```diff
@@ -1,16 +1,28 @@
 import { ScrollArea } from '@radix-ui/react-scroll-area'
 import { cn } from '../lib/utils'
 import * as AccordionPrimitive from '@radix-ui/react-accordion'
-import { NavArrowRight, NavArrowDown, Circle, CheckCircleSolid, XmarkCircleSolid } from '@harnessio/icons-noir'
+import {
+  NavArrowRight,
+  NavArrowDown,
+  Circle,
+  CheckCircleSolid,
+  XmarkCircleSolid,
+  Refresh,
+  ClockSolid
+} from '@harnessio/icons-noir'
 import React, { createContext, forwardRef, useCallback, useContext, useEffect, useState } from 'react'
 
+/**
+ * @TODO remove this from treeview component
+ */
 enum Status {
```

Comment:
move this to a types or interface file?

Reply:
I will eventually remove this from here in a follow up PR. This is currently duplicated with `ExecutionState` enum as well.